bobtemplates/plone/view/.mrbob.ini

```diff
@@ -0,0 +1,27 @@
+[questions]
+subtemplate_warning.question = Still want to run this subtemplate?
+subtemplate_warning.required = True
+subtemplate_warning.default = No
+subtemplate_warning.pre_ask_question = bobtemplates.plone.base:subtemplate_warning
+subtemplate_warning.post_ask_question = mrbob.hooks:validate_choices
+ bobtemplates.plone.base:subtemplate_warning_post_question
+subtemplate_warning.choices = Yes|No
+subtemplate_warning.choices_delimiter = |
+
+view_name.question = View Name
+view_name.required = True
+view_name.default = training
+view_name.help = Should be something like 'Training'
+
+title.question = Title
```

Comment:
For do you want to use the title here?
A view usually does not have a title but renders a title and description of some context.
In the first generated examples I would create a template which fill the title, description and content-core slots.
But title and description are comming from the current context.
One can always change this in the template by using static title and description instead of the dynamic context one.
No need to ask in mrbob for this.

Reply:
Actually, I was a bit confused about what exactly a view is. Going through the chapters in the Mastering Plone Development Tutorial, I was under the impression that a view is just a static webpage and missed out on the detail that it can be used to represent the details of a content type as context. So, I had added the `title` and `description` questions in the view template, thinking it to be an independent static webpage on its own. 
Now that I have that confusion cleared up, I'll make the required changes.

Also, since a content type is associated with a view should we add an option/flag `--create-view` in `plonecli add content_type` that also renders the view while rendering the content type?